text-ui-test/test7/test7.txt

```diff
@@ -0,0 +1,6 @@
+T//[✗]//td1
```

Comment:
There are a new attributes for tasks the priority 

For example for the todo task:
T//[✓]//td3//5

means the todo task have a priority of 5

Reply:
Otherwise, there will be a java.lang.ArrayIndexOutOfBoundsException when loading the data file